topics/proteomics/tutorials/neoantigen-4-peptide-verification/tutorial.md

```diff
@@ -0,0 +1,399 @@
+> > 2. The charge state recalculation is turned off because the data being processed is already assumed to have accurate charge assignments from the initial raw spectra. Recalculating the charge states could lead to unnecessary complications or errors in the context of this workflow, which focuses on peptide identification rather than charge state assignment.
+> >
+> {: .solution}
+>
+{: .question}
+
+## Peptide Validation with PepQuery2 with **PepQuery2**
+
+PepQuery2 is a tool used to validate novel peptides and proteins by searching mass spectrometry data against a reference protein database. In this workflow, PepQuery2 is used for peptide validation, specifically focusing on the detection of novel peptides or proteins. The input data includes a list of peptides (obtained from a previous step) and a protein reference database, which is also provided from the workflow history. The spectrum file, generated by msconvert, is used to search for peptide-spectrum matches (PSMs). The tool is configured with specific parameters, such as peptide charge and length, and the fragmentation method (CID/HCD). The output will help identify novel peptides that may not be present in the reference database, aiding in the discovery of potential new biomarkers or proteins.
+
+
+
+> <hands-on-title> PepQuery2 </hands-on-title>
+>
+> 1. {% tool [PepQuery2](toolshed.g2.bx.psu.edu/repos/galaxyp/pepquery2/pepquery2/2.0.2+galaxy2) %} with the following parameters:
+>    - *"Validation Task Type"*: `novel peptide/protein validation`
+>    - In *"Input Data"*:
+>        - *"Input Type"*: `peptide`
+>            - *"Peptides?"*: `Peptide list from your history`
+>                - {% icon param-file %} *"Peptide Sequences (.txt)"*: `NeoAntigen-Candidates.tabular` (Input dataset)
+>        - *"Protein Reference Database from"*: `history`
+>            - {% icon param-file %} *"Protein Reference Database File"*: `HUMAN_CRAP.fasta` (Input dataset)
+>        - *"MS/MS dataset to search"*: ` Spectrum Datasets from history`
+>            - {% icon param-file %} *"Spectrum File"*: `output` (output of **msconvert** {% icon tool %})
+>        - *"Report Spectrum Scan as"*: `spectrum title in MGF`
+>    - In *"Modifications"*:
+>        - *"Fixed modification(s)"*: ``
+>        - *"Variable modification(s)"*: ``
+>    - In *"Digestion"*:
+>        - *"Enzyme"*: `Non enzyme`
+>    - In *"Mass spectrometer"*:
+>        - In *"Tolerance"*:
+>            - *"Precursor Unit"*: `ppm`
+>        - In *"PSM"*:
+>            - *"Fragmentation Method"*: `CID/HCD`
+>            - *"Minimum Charge"*: `2`
+>            - *"Maximum Charge"*: `3`
+>            - *"Minimum length of peptide"*: `8`
+>            - *"Maximum length of peptide"*: `9`
+>    - *"Select outputs"*: ``
```

Comment:
should this have a value? please double-check (the workflow-to-tutorial script may miss some things).